src/platyPS/platyPS.psm1

```diff
@@ -637,7 +640,7 @@ function New-ExternalHelp
         }
 
         foreach ($group in $groups) {
-            $maml = GetMamlModelImpl ( $group.Group | ForEach-Object { Get-Content -Raw $_.FullName } )
+            $maml = GetMamlModelImpl ( $group.Group | ForEach-Object { Get-Content -Raw $_.FullName -Encoding $Encode} )
```

Comment:
We would need our own version to read the content. We are already do that for `Set-Content`, it would be similar